sippy-ng/src/component_readiness/ComponentReadiness.js

```diff
@@ -801,6 +807,22 @@ export default function ComponentReadiness(props) {
                           value={searchColumnRegex}
                           onChange={handleSearchColumnRegexChange}
                         />
+                        <Checkbox
+                          checked={redOnlyChecked}
+                          onChange={handleRedOnlyCheckboxChange}
+                          color="primary"
+                          size="small"
+                          style={{ borderRadius: 1 }}
+                        />
+                        <label
+                          htmlFor="redOnlyCheckbox"
+                          style={{
+                            textAlign: 'left',
+                            marginTop: 15,
+                          }}
+                        >
+                          Red Only
+                        </label>
```

Comment:
```suggestion
                        <FormControlLabel
                          control={
                            <Checkbox
                              checked={redOnlyChecked}
                              onChange={handleRedOnlyCheckboxChange}
                              color="primary"
                              size="small"
                              style={{ borderRadius: 1 }}
                            />
                          }
                          htmlFor="redOnlyCheckbox"
                          style={{
                            textAlign: 'left',
                            marginTop: 15,
                          }}
                          label="Red Only"
                        ></FormControlLabel>
```

Using the material UI label makes the alignment of the checkbox a little better

Old:
![image](https://github.com/openshift/sippy/assets/429763/606c6a42-b0a8-4c68-843a-5db4e2439583)


New:
![image](https://github.com/openshift/sippy/assets/429763/69a7a50c-34c9-4482-8c03-a8bc3ce9e936)

Reply:
Ah, definitely nicer.  Styling changes I tried could not get that text closer.

Reply:
will add import